File: Iniciante/URI 1037.lua
valor = tonumber(io.read())
if (valor >= 0 and valor <= 25) then
    print('Intervalo [0,25]')
elseif (valor > 25 and valor <= 50) then
    print('Intervalo (25,50]')
elseif (valor > 50 and valor <= 75) then
    print('Intervalo (50,75]')
elseif (valor > 75 and valor <= 100) then
    print('Intervalo (75,100]')
else
    print('Fora de intervalo')
end

File: Iniciante/URI 1037.php
<?php
    $valor = trim(fgets(STDIN));

    if ($valor >= 0 && $valor <= 25) {
       echo "Intervalo [0,25]\n";
    } else if ($valor > 25 && $valor <= 50) { 
       echo "Intervalo (25,50]\n";
    } else if ($valor > 50 && $valor <= 75) { 
       echo "Intervalo (50,75]\n";
    } else if ($valor > 75 && $valor <= 100) { 
       echo "Intervalo (75,100]\n";
    } else { 
       echo "Fora de intervalo\n";
    }
?>